### DISCUSSION TITLE
How many m/s can a human sneeze travel at maximum speed? 😤😷

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#907] [Galaxy Trivia] [answered]
**Bonus:** Where in the body does the sneezing process begin? What is the record for the longest sneeze?

**Comment by peterbaccaro** [comment, marked as answer]
The speed at which a human sneeze travels can vary, but it is generally estimated to be around **30 meters per second (m/s)** or 108 kilometers per hour (km/h). However, this speed can depend on various factors such as the individual's lung capacity, the force of the sneeze, and the environment in which the sneeze occurs.

The sneezing process typically begins in **the nasal cavity and upper respiratory tract**. When irritants, such as dust, pollen, or other foreign particles, stimulate the sensitive lining of the nasal passages, it triggers a reflex response to expel the irritants and clear the airway.

The **longest sneezing fit lasted 976 days** by Donna Griffiths (UK).
